docs/runbooks/CnaoNmstateMigration.md

```diff
@@ -1,14 +1,20 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # CnaoNmstateMigration
 
 ## Meaning
 
-Cluster-network-addons-operator (CNAO) will stop support of kubernetes-nmstate deployment.
-This alert fires when nmstate is deployed, but there is no Kubernetes NMState Operator installed.
+This alert fires when the `nmstate` API is deployed, but the Kubernetes NMState Operator is not installed.
+
+In a future update, the `Cluster-network-addons-operator` (CNAO) will stop supporting `kubernetes-nmstate` deployments. Instead of using the CNAO to deploy `kubernetes-nmstate`, it is recommended to install the stand-alone Kubernetes NMState Operator.
```

Comment:
The Cluster Network Addons Operator (CNAO) does not support `kubernetes-nmstate` deployments in OpenShift Virtualization 4.11 and later.

You can remove the 2nd sentence because it is repeated in the Mitigation